openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -142,3 +142,801 @@ class AfterTaxIncome(Variable):
+    definition_period = YEAR
+    documentation = """Lumpsum (or head) tax; appears as LSTAX variable in tc CLI minimal output"""
+
+
+class pre_c04600(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Personal exemption before phase-out"""
+
+
+class codtc_limited(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ptax_amc(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """Additional Medicare Tax from Form 8959 (included in payrolltax)"""
+    )
+
+
+class ptax_oasdi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Employee + employer OASDI FICA tax plus self-employment tax (excludes HI FICA so positive ptax_oasdi is less than ptax_was plus setax)"""
+
+
+class ptax_was(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Employee + employer OASDI + HI FICA tax"""
+
+
+class setax(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Self-employment tax"""
+
+
+class ymod(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ymod1(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """search taxcalc/calcfunctions.py for how calculated and used"""
+    )
+
+
+class ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Universal Basic Income benefit for filing unit"""
+
+
+class taxable_ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Amount of UBI benefit included in AGI"""
+
+
+class nontaxable_ubi(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Amount of UBI benefit excluded from AGI"""
+
+
+class mtr_paytax(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Marginal payroll tax rate (in percentage terms) on extra taxpayer earnings (e00200p)"""
+
+
+class mtr_inctax(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = """Marginal income tax rate (in percentage terms) on extra taxpayer earnings (e00200p)"""
+
+
+class aftertax_income(Variable):
+    value_type = float
+    entity = TaxUnit
+    definition_period = YEAR
+    documentation = (
+        """After tax income is equal to expanded_income minus combined"""
+    )
+
+
+class benefit_cost_total(Variable):
```

Comment:
I think we can drop this and `benefit_value_total`

Reply:
Yeah, done